packages/react-form-state/docs/building-forms.md

```diff
@@ -355,6 +355,46 @@ function MyComponent() {
 }
 ```
 
+### validateOnSubmit
+
+You can configure `<FormState />` to run all validators on the form before executing `onSubmit` by passing it the `validateOnSubmit` prop. If any of the validators return an error, the submit is cancelled and `onSubmit` is not run.
+
+```typescript
+import {TextField, Form, Button} from '@shopify/polaris';
+import FormState, {validators} from '@shopify/react-form-state';
+
+function MyComponent() {
+  return (
+    <FormState
```

Comment:
Do you need to actually include the `validateOnSubmit` prop here?

Reply:
lol, yes.